grouper/fe/template_util.py

```diff
@@ -11,17 +11,24 @@
 
 
 RESOURCE_INTEGRITY_VALUES = {}
+CHUNK_SIZE = 1024 * 16
```

Comment:
nit, but consider instead:
``` py
try:
    CHUNK_SIZE = os.statvfs(os.curdir).f_frsize
except AttributeError:
    # statvfs is unix only, fall back to sensible default
    CHUNK_SIZE = 4096 
```

fundamentally these files are tiny so it doesn't really matter.